### PR TITLE
Curator work list: Added scope, search field and ownership tab

### PR DESCRIPTION
**Pull request by garshol**
Added scope, search field and ownership tab

### CODE REVIEW

File: 1223-curator_worklist/357-curator_opens_my_worklist.feature [hunk 1]
```diff
@@ -5,10 +5,13 @@ Feature: Curator opens My Worklist
         Given that the user is logged in as Curator
         When they click the menu item My Worklist
         Then they see My Worklist page
+        And they see there designated institution scope
```
Comment (Eiriknil) [inline 1223-curator_worklist/357-curator_opens_my_worklist.feature:8]:
```suggestion
        And they see their designated institution scope
```

**Review by Eiriknil — DISMISSED** [review]

File: 1223-curator_worklist/357-curator_opens_my_worklist.feature (hunk 1 above)
Comment (elkebab) [inline 1223-curator_worklist/357-curator_opens_my_worklist.feature:8]: Stor bokstav for konsepter
```suggestion
        And they see there designated Institution scope
```

File: 1223-curator_worklist/357-curator_opens_my_worklist.feature [hunk 2]
```diff
@@ -43,3 +46,14 @@ Feature: Curator opens My Worklist
             | Submitter name     |
             | Date               |
         And the see that each Worklist Item has an Expand button
+
+    @2756
+    Scenario: Curator views Change Ownweship tab
```
Comment (elkebab) [inline 1223-curator_worklist/357-curator_opens_my_worklist.feature:51]:
```suggestion
    Scenario: Curator views Change Ownership tab
```

File: 1223-curator_worklist/357-curator_opens_my_worklist.feature [hunk 3]
```diff
@@ -43,3 +46,14 @@ Feature: Curator opens My Worklist
...
+        Given that the Curator opens My Worklist
+        When they click the tab "Change ownership"
+        Then they see a list of Worklist Items with fields:
+            | Request status     |
+            | Registration title |
+            | Submitter name     |
+            | Date               |
+        And the see that each Worklist Item has an Expand button
```
Comment (elkebab) [inline 1223-curator_worklist/357-curator_opens_my_worklist.feature:59]: mangler det et scenario hvor man faktisk bytter eier her?

**Review by elkebab — APPROVED** [review]

**Review by terjehell — APPROVED** [review]

**Pull request merged**